src/LoginServer/World/Character.cs

```diff
@@ -64,5 +65,33 @@ public int[] GetEquipmentProperties()
 			// TODO: This needs to return the actual properties of equipment which have variable lengths.
 			return this.GetEquipIds();
 		}
+
+		/// <summary>
+		/// Sets the default equipment for a character.
+		/// </summary>
+		/// <returns></returns>
+		public bool SetDefaultEquipment()
+		{
+			var jobData = LoginServer.Instance.Data.JobDb.Find(this.Job);
+			if (jobData == null)
+			{
+				Log.Error("SetDefaultEquipment : Unable to set default equipment for job '{0}' because it doesn't exist.", this.Job);
```

Comment:
I don't really like logging in the domain like this. Perhaps there is a nicer way.

Reply:
You could always make the method take a list of items that you prepare in the packet handler.